cmd/cstor-pool-mgmt/volumereplica/volumereplica_test.go

```diff
@@ -221,6 +245,12 @@ func TestStatusHelperProcess(*testing.T) {
 	if os.Getenv("StatusType") == ZfsStatusRebuilding {
 		fmt.Fprint(os.Stdout, mockedStatusOutputRebuilding)
 	}
+	if os.Getenv("StatusType") == "Reconstructing" {
+		fmt.Fprint(os.Stdout, mockedStatusOutputReconstructing)
+	}
+	if os.Getenv("StatusType") == "NonQuorumDegraded" {
```

Comment:
string literals are used at multiple places

Reply:
Since it is a test file used as literals. I will update the test structure

Reply:
Updated